### PR TITLE
Revert "Delete SHA256SUMS for now"

### PR DESCRIPTION
**Pull request by sw**
Reverts ggerganov/llama.cpp#416

### CODE REVIEW

**Comment by prusnak** [comment]
Let's drop lines with `ggml-model-*.bin` from the file, since these are generated and not downloaded.

File: SHA256SUMS [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+135c563f6b3938114458183afb01adc9a63bef3d8ff7cccc3977e5d3664ecafe  models/65B/consolidated.00.pth
+9a600b37b19d38c7e43809485f70d17d1dc12206c07efa83bc72bb498a568bde  models/65B/consolidated.01.pth
+e7babf7c5606f165a3756f527cb0fedc4f83e67ef1290391e52fb1cce5f26770  models/65B/consolidated.02.pth
+73176ffb426b40482f2aa67ae1217ef79fbbd1fff5482bae5060cdc5a24ab70e  models/65B/consolidated.03.pth
+882e6431d0b08a8bc66261a0d3607da21cbaeafa96a24e7e59777632dbdac225  models/65B/consolidated.04.pth
+a287c0dfe49081626567c7fe87f74cce5831f58e459b427b5e05567641f47b78  models/65B/consolidated.05.pth
+72b4eba67a1a3b18cb67a85b70f8f1640caae9b40033ea943fb166bd80a7b36b  models/65B/consolidated.06.pth
+d27f5b0677d7ff129ceacd73fd461c4d06910ad7787cf217b249948c3f3bc638  models/65B/consolidated.07.pth
+999ed1659b469ccc2a941714c0a9656fa571d17c9f7c8c7589817ca90edef51b  models/65B/params.json
+36079249f53c292a4c2302d7784005dcae94c865f0bedfdbfa51d9ddad402935  models/alpaca-30B/params.json
```
Comment (Green-Sky) [inline SHA256SUMS:20]: i don't think we need the alpaca params.json

**Review by Green-Sky — CHANGES_REQUESTED** [review]

File: SHA256SUMS (hunk 1 above)
Comment (prusnak) [inline SHA256SUMS:20]: Maybe also remove this last line. It seems odd to just keep this one.

**Review by prusnak — APPROVED** [review]

**Comment by prusnak** [comment]
> Assuming the generation is deterministic, we'll still eventually want to have them in here, right?

I think so. once the file format stop changing every couple of days :-)

**Comment by prusnak** [comment]
I added one commit to reflect changes in the readme and also added `models/tokenizer.model` to `SHA256SUMS`

**Review by prusnak — APPROVED** [review]

**Review by Green-Sky — APPROVED** [review]
did not check the hashes, but looks good

**Pull request merged**